Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -516,6 +516,9 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
 
             try tokenStore.store(newAuthToken)
         case .useExisting:
+            guard try tokenStore.fetchToken() != nil else {
+                throw TunnelError.startingTunnelWithoutAuthToken
+            }
```

Comment:
If we plan to use the existing auth token we might as well make sure there is one.